### PR TITLE
[DDCNL-4120][SM]: a11y Issue: None of the links warn the user about opening a new…

### PR DESCRIPTION
**Pull request by sureshhmrc**
… tab (AAA)

https://docs.google.com/document/d/1T6acDbdIhr9uaMItvmqbdt01kP73BE1B0YT5oKShY4I/edit#heading=h.8np0ll8vi9l

### CODE REVIEW

File: app/views/components/anchor.scala.html [hunk 1]
```diff
@@ -17,12 +17,12 @@
...
 )(implicit messages: Messages)
 
-<a
-  href="@href"
-  @target.map { target =>
-    target="@target"
-  }
-  >@messages(messageKey)</a>
+@if(opensInNewTab) {
+    <a href="@href" class="govuk-link" rel="noreferrer noopener" target="_blank">@messages(messageKey) @messages("link.opens.new.tab.text")</a>
```
Comment (Muhammad-Ali-Ahmad) [inline app/views/components/anchor.scala.html:25]: what does class "govuk-link" do?

Comment (sureshhmrc) [reply app/views/components/anchor.scala.html:25]: this is the styling used in the design pages : https://design-system.service.gov.uk/styles/typography/#links 
we have always used it for anchors, so copy-pasted it really :)

Comment (sureshhmrc) [reply app/views/components/anchor.scala.html:25]: I am happy to remove it if you think its not required here

Comment (Muhammad-Ali-Ahmad) [reply app/views/components/anchor.scala.html:25]: Until we move to hmrc frontend, the govuk design system styles are irrelevant as we dont yet have those styles. Currently all our services use assets frontend (older library for frontend assets)
You can remove the class for now

Comment (sureshhmrc) [reply app/views/components/anchor.scala.html]: removed it

**Review by Muhammad-Ali-Ahmad — APPROVED** [review]

**Pull request merged**